components/core/src/clp/clo/CommandLineArguments.cpp

```diff
@@ -179,12 +179,15 @@ auto CommandLineArguments::parse_ir_extraction_arguments(
     // Define IR extraction options
     po::options_description options_ir_extraction("IR Extraction Options");
     // clang-format off
-    options_ir_extraction
-            .add_options()(
-                    "target-size",
-                    po::value<size_t>(&m_ir_target_size)->value_name("SIZE"),
-                    "Target size (B) for each IR chunk before a new chunk is created"
-            );
+    options_ir_extraction.add_options()(
+            "target-size",
+            po::value<size_t>(&m_ir_target_size)->value_name("SIZE"),
```

Comment:
```suggestion
            po::value<size_t>(&m_ir_target_size)
                    ->value_name("SIZE")
                    ->default_value(m_ir_target_size),
```
Probably want a default value?

Reply:
ditto